frontend/apps/reader/modules/readerview.lua

```diff
@@ -815,11 +815,29 @@ end
 
 function ReaderView:onSetRotationMode(rotation)
     if rotation ~= nil then
-        if rotation == Screen:getRotationMode() then
+        if rotation == Screen:getRotationMode() then -- no change
             return true
         end
+
+        local requested_screen_mode -- landscape or portrait
+        if rotation == Screen.DEVICE_ROTATED_UPRIGHT or rotation == Screen.DEVICE_ROTATED_UPSIDE_DOWN then
+            requested_screen_mode = "portrait"
+        else
+            requested_screen_mode = "landscape"
+        end
```

Comment:
Since you like maths, `if bit.band(rotation, 1) == 0 then portrait else landscape end` also works ;).

(LinuxFB constants, portraits are odd, landscapes are even).

Reply:
Modulo is super slow, though, which is why I prefer an AND ;).

But, yeah, this is perfectly harmless, I just like the odd/even thing ;).